CHANGELOG.develop

```diff
@@ -892,6 +892,7 @@ Other:
     - ~G~ go to last candidate
     (thanks to duianto)
 - Fixed:
+  - Fixed agda layer break after Agda 2.6.0.0 api changes
```

Comment:
Move this line to a new `Agda` subsection in the section:
https://github.com/syl20bnr/spacemacs/blob/develop/CHANGELOG.develop#layer-changes-and-fixes